cmake/opentelemetry-cpp-config.cmake.in

```diff
@@ -70,7 +70,39 @@ set(OPENTELEMETRY_VERSION
 
 # ##############################################################################
 
-find_package(Threads)
+include(CMakeFindDependencyMacro)
+
+find_dependency(Threads)
+
+if(@WITH_ABSEIL@)
+  find_dependency(absl)
+endif()
+
+if(@WITH_OTLP_GRPC@)
+  find_dependency(gRPC)
+endif()
+
+if("@OpenTracing_FOUND@")
```

Comment:
Why are double quotes needed here but not for the above variable like `WITH_OTLP_GRPC`? Is it because the variable here could be undefined?

Reply:
These vars may not be set and `@VAR@` will be replaced by nothing, which has syntax error.